README.md

```diff
@@ -150,6 +150,31 @@ $writer->addRowsWithStyle($multipleRows, $style); // style will be applied to al
 $writer->close();
 ```
 
+Adding borders to a row requires a ```Border``` object.
+
+```php
+use Box\Spout\Common\Type;
+use Box\Spout\Writer\WriterFactory;
+use Box\Spout\Writer\Style\StyleBuilder;
+use Box\Spout\Writer\Style\Border;
+use Box\Spout\Writer\Style\BorderPart;
+
+$border = new Border();
+$border
+    ->addPart(new BorderPart(Border::BOTTOM, Border::STYLE_SOLID, Border::WIDTH_THICK, Color::ORANGE))
```

Comment:
I would keep BorderPart internal and expose user friendly APIs on the Border object.

```php
$borders = new BordersBuilder()
    ->setBorderBottom(Border::STYLE_SOLID, Color::ORANGE, Border::WIDTH_THICK); // I re-ordered the properties by order of potential usage
    ->setBorderRight(Border::STYLE_DASHED, Color::GREEN, Border::WIDTH_THIN);
    ->build();

$style = (new StyleBuilder())
    ->setBorders($borders)
    ->build();
```